components/automate-cli/pkg/verifyserver/server/api/v1/status_test.go

```diff
@@ -0,0 +1,80 @@
+package v1_test
+
+import (
+	"io/ioutil"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/logger"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/models"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/server"
+	v1 "github.com/chef/automate/components/automate-cli/pkg/verifyserver/server/api/v1"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/services/statusservice"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/utils/fiberutils"
+	"github.com/gofiber/fiber"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func SetupMockStatusService() statusservice.IStatusService {
+	return &statusservice.MockStatusService{
+		GetServicesFunc: func() []models.ServiceDetails {
+			return []models.ServiceDetails{
+				{
+					ServiceName: "automate",
+					Version:     "4.5.20",
+					Status:      "healthy",
```

Comment:
Is the message going to come healthy or ok or something else, please check in hab svc status

Reply:
It will be up/down. i have changed it.